topics/admin/tutorials/maintenance/slides.html

```diff
@@ -0,0 +1,95 @@
+---
+layout: tutorial_slides
+logo: "GTN"
+
+title: "Server Maintenance"
+zenodo_link: ""
+questions:
+  - How to maintain a Galaxy service?
+objectives:
+  - Learn about different maintenance steps
+time_estimation: "1h"
+key_points:
+  - Remember to back Galaxy up
+  - Use Ansible
+contributors:
+  - natefoo
+  - bgruening
+  - slugger70
+  - erasche
+---
+
+# Runaway Storage
+
+Tips:
+- Set quotas
+- `tmpwatch` your job working directory
+  - `cleanup_job` in `galaxy.yml` (defaults to `always` though)
+- Set up dataset cleanup
+
+---
+# Dataset Cleanup
+
+- `scripts/cleanup_datasets/pgcleanup.py`: PostgreSQL-optimized fast cleanup script
+- `scripts/cleanup_datasets/cleanup_datasets.py`: General cleanup script
+- `gxadmin cleanup 30`: calls pgcleanup
+
+---
+# Dataset Cleanup Lifecycle
+
+Mark deleted all "anonymous" histories not used within the last `$days` days:
+
+`python ./scripts/cleanup_datasets/pgcleanup.py -o $days -s delete_userless_histories`
+
+Remove all history exports older than `$days` days:
+
+`python ./scripts/cleanup_datasets/pgcleanup.py -o $days -s delete_exported_histories`
+
+Mark purged HDAs in histories deleted `$days` or more days ago (not user-recoverable):
+
+`python ./scripts/cleanup_datasets/pgcleanup.py -o $days -s purge_deleted_histories`
+
+---
+# Dataset Cleanup Lifecycle
+
+Mark purged individual HDAs deleted `$days` or more days ago (not user-recoverable):
+
+`python ./scripts/cleanup_datasets/pgcleanup.py -o $days -s purge_deleted_hdas`
+
+Mark datasets with all purged HDAs last updated `$days` or more days ago deleted:
+
+`python ./scripts/cleanup_datasets/pgcleanup.py -o $days -s delete_datasets`
+
+Mark purged all datasets last updated `$days` or more days ago **and remove from disk**:
+
+`python ./scripts/cleanup_datasets/pgcleanup.py -o $days -s purge_datasets`
+
+---
+# Nagios
+
+General purpose tool for monitoring systems and services.
+
+Galaxy-specific check in `contrib/nagios/`: Runs Galaxy jobs
+
+---
+
+# Sentry
+
+* Motto: *"Stop hoping your users will report errors"*
+* Error tracking and analysing tool.
+* Galaxy has Sentry middleware that you can enable in configuration.
```

Comment:
I think Nagios and probably also Sentry should be moved to the monitoring slide deck, but I'm not using either yet, so feel free to ignore.